cmd/probe.go

```diff
@@ -103,7 +103,12 @@ func runProbe(cmd *cobra.Command, args []string) {
 	leaderElectionClient := kubernetes.NewForConfigOrDie(rest.AddUserAgent(config, "dependency-watchdog-election"))
 	recorder := createRecorder(leaderElectionClient)
 	run := func(ctx context.Context) {
-		go serveMetrics()
+		go func() {
+			err := serveMetrics()
+			if err != nil {
+				klog.Fatalf("Can't serve metrics: %s", err)
```

Comment:
Should this kill the container? Metrics are good to have and not must to have, right?

Reply:
okay will error on this